hera_pspec/pspecdata.py

```diff
@@ -1597,6 +1598,45 @@ def rephase_to_dset(self, dset_index=0, inplace=True):
         if inplace is False:
             return dsets
 
+    def Jy_to_mK(self, beam=None):
```

Comment:
Should the example Jupyter notebook be changed to use this method now?

Reply:
yup

Reply:
except that to run on our test data another small bug fix is required that is addressed in a separate PR open right now. I'll update the notebook at a (near) future date incorporating all the changes we are making in this and a few other simultaneous PRs.